### PR TITLE
[Feat] overview recommendations for BPMN

### PR DESCRIPTION
**Pull request by MattDodsonEnglish**
Lightly works on #87 as it provides a bit more context to datasources

### CODE REVIEW

**Comment by MattDodsonEnglish** [comment]
How the diagram renders:


image

File: content/how-to/bpmn/orchestration-overview.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+description: >
+   An overview of how to use Rhize's custom BPMN engine and UI to orchestrate workflows.
+weight: 10
+menu:
+  main:
+    parent: howto-bpmn
+---
+
+Rhize's BPMN engine and UI provides a way to coordinate complex tasks and exchange messages between the Rhize platform and your internal services.
+For example, you can use BPMN to coordinate messages between services, create alerts and events, transform incoming work orders into their ISA-95 representation, and store them in their Rhize DB.
```
Comment (Jarrah-libremfg) [inline content/how-to/bpmn/orchestration-overview.md:13]:
```suggestion
For example, you can use BPMN to coordinate messages between services, create alerts and events, transform incoming work orders into their ISA-95 representation, and store them in the Rhize DB.
```

File: content/how-to/bpmn/orchestration-overview.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+     - `createAndRunBpmn` starts a workflow and does not wait for the response (asynchronous).
+     - `createAndRunBPMNSync` starts a workflow and waits for the process to complete or abort (synchronous).
+
+## Data processing and API calls
+
+Besides exchanging messages, BPMN workflows can also transform and evaluate payloads and make HTTP requests.
+All the data in a process is stored in a {{< abbr "process variable context" >}} that each element can access.
+
+The general logical elements are as follows:
+- To evaluate conditions, use a [gateway]({{< relref "/how-to/bpmn/bpmn-elements#gateways" >}})
```
Comment (Jarrah-libremfg) [inline content/how-to/bpmn/orchestration-overview.md:75]:
```suggestion
- To evaluate conditions, use an [exclusive gateway]({{< relref "/how-to/bpmn/bpmn-elements#gateways" >}})
```

Gateways allow more than just conditionals, so just adjusting this to be a bit more specific.

**Review by Jarrah-libremfg — APPROVED** [review]
Looks good to me. Just two minor wording changes.

**Pull request merged**